### PR TITLE
bytes 2 + 4

### PR DESCRIPTION
**Pull request by Thomas-Smets**
Separate function makes more sense when https://github.com/arcadia-finance/accounts-v2/pull/128 is merged as well.

### CODE REVIEW

File: src/accounts/AccountV1.sol [hunk 1]
```diff
@@ -821,10 +821,9 @@ contract AccountV1 is AccountStorageV1, IAccount {
             IRegistry(registry).batchProcessDeposit(creditor, assetAddresses, assetIds, assetAmounts);
 
         for (uint256 i; i < assetAddresses.length; ++i) {
-            if (assetAmounts[i] == 0) {
-                // Skip if amount is 0 to prevent storing addresses that have 0 balance.
-                continue;
-            }
+            // Skip if amount is 0 to prevent storing addresses that have 0 balance,
+            // these are also skipped for batchProcessDeposit in the Registry.
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:825]:
```suggestion
            // these are also skipped during batchProcessDeposit in the Registry.
```

File: src/accounts/AccountV1.sol [hunk 2]
```diff
@@ -894,10 +893,9 @@ contract AccountV1 is AccountStorageV1, IAccount {
             IRegistry(registry).batchProcessWithdrawal(creditor, assetAddresses, assetIds, assetAmounts);
 
         for (uint256 i; i < assetAddresses.length; ++i) {
-            if (assetAmounts[i] == 0) {
-                // Skip if amount is 0 to prevent transferring 0 balances.
-                continue;
-            }
+            // Skip if amount is 0 to prevent transferring 0 balances,
+            // these are also skipped for batchProcessWithdrawal in the Registry.
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:897]:
```suggestion
            // these are also skipped during batchProcessWithdrawal in the Registry.
```

**Review by j-vp — APPROVED** [review]
lgtm

**Review by acollette — APPROVED** [review]
gtg

File: src/asset-modules/interfaces/IRegistry.sol [hunk 3]
```diff
@@ -86,14 +86,14 @@ interface IRegistry {
     function getRateInUsd(bytes32 oracleSequence) external view returns (uint256);
 
     /**
-     * @notice Calculates the usd value of an asset.
+     * @notice Calculates the USD-values of underlying assets.
```
Comment (zekiblue) [inline src/asset-modules/interfaces/IRegistry.sol:89]:
```suggestion
     * @notice Calculates the USD values of underlying assets.
```

**Review by zekiblue — COMMENTED** [review]
lgtm

**Pull request merged**